SF_Users_API/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I am not sure we need a new package.json here.  SHouldn't this just be a part of the existing application structure?